client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -588,6 +588,9 @@ function getFallbackDestination( {
 		const emails = titanProducts[ 0 ].extra?.email_users;
 		if ( emails && emails.length > 0 ) {
 			debug( 'site with titan products' );
+			if ( cart?.products?.length === 1 ) {
+				return `/mailboxes/${ siteSlug }?new-email=${ emails[ 0 ].email }`;
```

Comment:
Is it possible to add a test for this change?

https://github.com/Automattic/wp-calypso/blob/81e315e20839cc959067479087c1f22e32a07643/client/my-sites/checkout/get-thank-you-page-url/index.ts#L2-L7

Reply:
Added